### DISCUSSION TITLE
Alterar a versão .NET

### DISCUSSION THREAD

**Discussion by leoolivie** [balta-io/forum#47] [Tira Dúvidas] [answered]
Como faço para alterar a versão do .NET do 6.0 para o 2.0 no VSCODE?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @leoolivie, como vai?

Você pode especificar a versão do Framework na hora de criar a aplicação utilizando o flag `-f`.
```
dotnet new console -o MeuApp -f net2.0
```

Caso já tenha criado o App, você precisa ir no arquivo `.csproj` e trocar o `TargetFramework` para `.net2.0`.
Fazendo isto, você precisa fazer o downgrade dos pacotes caso hajam...
